### PR TITLE
[FLINK-23484][benchmark] Supports base benchmark for ChangelogStateBackend

### PR DESCRIPTION
**Pull request by masteryhx**
## What is the purpose of the change

- Supports ChanglogStateBackend Benchmark. Will also prepare a related pr in flink-benchmarks repo.

## Brief change log

- Move benchmark utils from rocksdb repo to flink-test-util.
- Add Changelog supports in StateBenchmarkBase.

## Verifying this change

This change is already covered by existing tests, such as `StateBackendBenchmarkUtilsTest`.

## Does this pull request potentially affect one of the following parts:

  - Dependencies (does it add or upgrade a dependency): no
  - The public API, i.e., is any changed class annotated with `@Public(Evolving)`: no
  - The serializers: no
  - The runtime per-record code paths (performance sensitive): no
  - Anything that affects deployment or recovery: JobManager (and its components), Checkpointing, Kubernetes/Yarn, ZooKeeper: no
  - The S3 file system connector: no

## Documentation

  - Does this pull request introduce a new feature? no
  - If yes, how is the feature documented? no

### CODE REVIEW

**Comment by masteryhx** [comment]
@flinkbot run azure

**Comment by masteryhx** [comment]
@Myasuka @fredia Could you help to take a review ?
This pr should be reviewed together: https://github.com/apache/flink-benchmarks/pull/75

File: flink-test-utils-parent/flink-test-utils/pom.xml [hunk 1]
```diff
@@ -133,6 +133,19 @@ under the License.
...
+			<groupId>org.apache.flink</groupId>
+			<artifactId>flink-streaming-java</artifactId>
+			<version>${project.version}</version>
+			<type>test-jar</type>
+		</dependency>
+
+		<dependency>
+			<groupId>org.apache.flink</groupId>
+			<artifactId>flink-statebackend-rocksdb</artifactId>
+			<version>${project.version}</version>
```
Comment (fredia) [inline flink-test-utils-parent/flink-test-utils/pom.xml:146]: Is `<scope>runtime</scope>` required？

File: flink-test-utils-parent/flink-test-utils/src/test/java/org/apache/flink/state/benchmark/RescalingBenchmark.java [hunk 2]
```diff
@@ -16,7 +16,7 @@
  * limitations under the License
  */
 
-package org.apache.flink.contrib.streaming.state.benchmark;
+package org.apache.flink.state.benchmark;
```
Comment (fredia) [inline flink-test-utils-parent/flink-test-utils/src/test/java/org/apache/flink/state/benchmark/RescalingBenchmark.java:19]: Have you prepared the changes on the flink-benchmarks side? we should minimize the time flink-benchmarks compilation fails.

Comment (masteryhx) [reply flink-test-utils-parent/flink-test-utils/src/test/java/org/apache/flink/state/benchmark/RescalingBenchmark.java:19]: Sure. PR Link: https://github.com/apache/flink-benchmarks/pull/75

**Review by Myasuka — APPROVED** [review]
LGTM, thanks for creating this PR.

**Pull request merged**